src/SWP/Bundle/ContentBundle/Controller/RelatedArticleController.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Superdesk Web Publisher Content Bundle.
+ *
+ * Copyright 2019 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2019 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\ContentBundle\Controller;
+
+use Nelmio\ApiDocBundle\Annotation\ApiDoc;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use SWP\Bundle\ContentBundle\Model\ArticleInterface;
+use SWP\Component\Common\Criteria\Criteria;
+use SWP\Component\Common\Exception\NotFoundHttpException;
+use SWP\Component\Common\Pagination\PaginationData;
+use SWP\Component\Common\Response\ResourcesListResponse;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+
+class RelatedArticleController extends Controller
+{
+    /**
+     * List related articles.
+     *
+     * @ApiDoc(
+     *     resource=true,
+     *     description="List related articles",
+     *     statusCodes={
+     *         200="Returned on success.",
+     *         500="Unexpected error."
+     *     },
+     *     filters={
+     *         {"name"="sorting", "dataType"="string", "pattern"="[updatedAt]=asc|desc"}
+     *     }
+     * )
+     * @Route("/api/{version}/content/related/articles/{id}", options={"expose"=true}, defaults={"version"="v1"}, name="swp_api_content_list_related_articles", requirements={"id"="\d"})
```

Comment:
Why not `/api/{version}/content/articles/{id}/related` ? Imo will be more consistent with with current articles endpoint.

Reply:
because there is already a route `/api/{version}/content/articles/{id}` where id param has `.+` requirement so route `api/v1/content/articles/7/related` is matched by `/api/{version}/content/articles/{id}` and not `/api/{version}/content/articles/{id}/related`. We should either limit id in `api/{version}/content/articles/{id}` route to `\d` or manually configure route in yaml and set `/api/{version}/content/articles/{id}/related` route to be mached before `api/{version}/content/articles/{id}`. But since we use annotations we should stick with that, otherwise we move alll routing definition to yaml files IMO.

Reply:
In case of annotation order of actions in controller set's their priority - this same like in yaml. If action will be before normall article get then it will be checked first. That should be enough.

Reply:
but this is a separate controller and routes priorities are set based on controller order by name

Reply:
I can move this action to ArticleController but it makes sense to keep it in a separate one to not mix stuff IMO